extension/ezoe/modules/ezoe/upload.php

```diff
@@ -167,10 +174,23 @@
                         $attr->store();
                         break;
                     case 'ezimage':
-                        // validation has been done by eZContentUpload
                         $content = $attr->attribute( 'content' );
-                        $content->setAttribute( 'alternative_text', $postVar );
-                        $content->store( $attr );
+
+                        // Check if the alt text is required
+                        if(
+                            !trim( $postVar ) &&
```

Comment:
OK - I changed up the order of the conditions, but I **_REALLY_** want to use eZImageType::validateImageAltText()